heasarc/test/test_all.sh

```diff
@@ -7,9 +7,9 @@
 set -e
 
 xte=true
-ciao=true
-fermi=true
-xmm=true
+ciao=false
+fermi=false
+xmm=false
```

Comment:
I never got these working fully automatically, but it looks like you might have?